emulators/qemu/Portfile

```diff
@@ -260,6 +254,14 @@ variant ssh description {Support remote block devices over SSH} {
     depends_lib-append      port:libssh
 }
 
+variant dbus description {Export the VM display through D-Bus} {
+    configure.args-replace  --disable-dbus-display --enable-dbus-display
+}
+
+variant virtfs description {Enable 9pfs support} {
+    configure.args-replace  --disable-virtfs --enable-virtfs
+}
```

Comment:
This does not add any additional dependencies. Therefore I do not think we need to hide it in a variant. Let's just always enable this feature.

Reply:
Hi @raimue,

I have made this change.

Thanks,

-ranga